editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -739,18 +740,40 @@ export const NavigatorItem: React.FunctionComponent<
     [dispatch, navigatorEntry.elementPath],
   )
 
+  const canBeSelected = useEditorState(
+    Substores.metadata,
+    (store) => {
+      const element = MetadataUtils.findElementByElementPath(
+        store.editor.jsxMetadata,
+        navigatorEntry.elementPath,
+      )
+
+      const isHTMLTagElement =
+        element != null &&
+        isRight(element.element) &&
+        isJSXElement(element.element.value) &&
+        element.element.value.name.baseVariable === 'html'
+
+      return !isHTMLTagElement
+    },
+    'NavigatorItem canBeSelected',
+  )
```

Comment:
Since we already have the functionality, we should be using the locking mechanism for this. You can add a check into `updateSimpleLocks` <https://github.com/concrete-utopia/utopia/blob/master/editor/src/core/shared/element-locking.ts#L8-L33>